### PR TITLE
[radio] update min receive on ahead/after values

### PR DESCRIPTION
**Pull request by edmont**

### CODE REVIEW

File: src/nrf52811/openthread-core-nrf52811-config.h [hunk 1]
```diff
@@ -247,14 +247,27 @@
...
+ * @def OPENTHREAD_CONFIG_MIN_RECEIVE_ON_AFTER
+ *
+ * The minimum time (in microseconds) after the MHR start that the radio should be in receive state in order
+ * to properly receive any IEEE 802.15.4 frame. Defaults to the duration of a maximum size frame, plus AIFS,
+ * plus the duration of maximum enh-ack frame. Platforms are encouraged to improve this value for energy
+ * efficiency purposes.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_MIN_RECEIVE_ON_AFTER
+#define OPENTHREAD_CONFIG_MIN_RECEIVE_ON_AFTER 0
```
[the remark this answers is not in the document]
Comment (zhanglongxia) [reply src/nrf52811/openthread-core-nrf52811-config.h:270]: +1

[the remark this answers is not in the document]
Comment (edmont) [reply src/nrf52811/openthread-core-nrf52811-config.h:270]: Hi, nRF radio will automatically extend the duration once the SHR has been detected. And 40 us of the preamble before the SFD will be enough to detect it. That's why a minimum of 104 us is required ahead of the MHR : Preamble (last 40 us) + SFD (32 us) + PHR (32 us). And not time is needed after due to the automatic extension. This allows to save some energy whenever there is nothing to be received.

[the remark this answers is not in the document]
Comment (zhanglongxia) [reply src/nrf52811/openthread-core-nrf52811-config.h:270]: Could you add the reason to the comment of the definition? This is a great help for developers to know why the value should be set up like this.

[the remark this answers is not in the document]
Comment (edmont) [reply src/nrf52811/openthread-core-nrf52811-config.h:273]: Done.

**Review by zhanglongxia — APPROVED** [review]
LGTM

**Review by Irving-cl — APPROVED** [review]
LGTM

**Pull request merged**